src/App.jsx

```diff
@@ -1,44 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
+import { Tabs } from './components/Tabs/Tabs';
+
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
+export const App = () => {
+  const [activeTabId, onTabSelected] = useState(tabs[0].id);
```

Comment:
The `useState` hook returns an array with two elements: the current state and a function to update it. Here, `onTabSelected` should be the function to update the state, not the event handler itself. You should rename `onTabSelected` to something like `setActiveTabId` to reflect its purpose and ensure it is used correctly.